src/components/Filters/SchemaSieve.spec.ts

```diff
@@ -782,28 +802,34 @@ describe('SchemaSieve', () => {
 					'Entry 3': {
 						category: ['Seed'],
 					},
-					'Entry 4': {
-						category: null,
-					},
-					'Entry 5': {
-						foo: 'bar',
-					},
+					// 'Entry 4': {
+					// 	category: null,
+					// },
+					// 'Entry 5': {
+					// 	foo: 'bar',
+					// },
```

Comment:
Should we re-enable these and leave a comment explaining that not_contains needs the filtered prop to exist and be a string at the first place?